app/controllers/api/v1/organisations_users_controller.rb

```diff
@@ -27,7 +26,12 @@ class OrganisationsUsersController < Api::V1::ApiController
       api :POST, "/v1/organisations_user", "Create a package"
       param_group :organisations_user
       def create
-        save_and_render_object_with_errors(@organisations_user)
+        builder = OrganisationsUserBuilder.new(params['organisations_user']).build
+        if builder[:result]
+          save_and_render_object_with_errors(builder[:organisations_user])
```

Comment:
be good to use builder['result'] - string instead of symbols... are we using `hash_with_indifferent_access`? Same for `builder[:organisations_user]` etc etc